code/game/gamemodes/blob/overmind.dm

```diff
@@ -224,3 +241,69 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 			return 0
 		loc = NewLoc
 		return 1
+
+/mob/camera/blob/mind_initialize()
+	. = ..()
+	var/datum/antagonist/blob/B = mind.has_antag_datum(/datum/antagonist/blob)
+	if(!B)
+		mind.add_antag_datum(/datum/antagonist/blob)
+
+/datum/antagonist/blob
```

Comment:
move the declaration and procs to a new file in [here](https://github.com/tgstation/tgstation/tree/master/code/datums/antagonists)